tests/Js/Main/ListCollectorTests.fs

```diff
@@ -0,0 +1,26 @@
+module Fable.Tests.ListCollector
+
+open Util.Testing
+open Fable.Tests.Util
+open Microsoft.FSharp.Core.CompilerServices
+
+let cutOffLast list =
+    let mutable headList = ListCollector<'a>()
+
+    let rec visit list =
+        match list with
+        | []
+        | [ _ ] -> ()
+        | head :: tail ->
+            headList.Add(head)
+            visit tail
+
+    visit list
+    headList.Close()
+
+let tests =
+  testList "ListCollector" [
+    testCase "ListCollector.Add and .Close" <| fun () ->
+        let result = cutOffLast [ 1; 2; 3 ]
+        result |> equal [ 1; 2 ]
+  ]
```

Comment:
Can we add tests for each `method` available and also edges cases like calling `.close()` on an empty `ListCollector`?

Reply:
I don't agree yes for this module it is probably enough, but it happens several times that doing so allowed us to catch behaviour we were not aware of.

There are still some APIs in fable-library that don't behave correctly because they didn't have a test associated for every usage. Having tests also allows us to catch on regressions.

I added the tests to cover the most common cases.